docs/semgrep-app/dashboard.md

```diff
@@ -42,6 +42,15 @@ To change the time range of scan data over time:
 1. Click the **Last 1 month** button.
 2. Select a time range from the drop-down box. The Dashboard, including all widgets, reloads to reflect data from the selected time period.
 
+## Filtering through findings from projects
+
+The Dashboard displays data from scans for **all of the organization's projects** by default. You can select one or a few projects to filter the dashboard widgets to only reflect scans from selected projects. Selecting a few projects gives you a more targeted view of those projects' security posture.
+
+To change the projects filter:
+
+1. Click the **All projects** button.
+2. Select the project(s) from the drop-down box. The Dashboard, including all widgets, reloads to reflect data from the selected project(s).
```

Comment:
I was not sure here because of: https://developers.google.com/style/word-list?hl=en#drop-down

```suggestion
2. Select the project(s) from the drop-down menu. The Dashboard, including all widgets, reloads to reflect data from the selected project(s).
```

Reply:
I copied "box" from the section above, though I agree that `drop-down menu` makes more sense? I'm just going to keep box for the sake of consistency if that's ok.